c3d/src/C3d.cpp

```diff
@@ -90,6 +174,7 @@ class C3dSimpleSpelling: public ParsedAttrInfo {
   }
 };
 
+std::vector<c3d_var_info> C3dSimpleSpelling::c3d_scope;
```

Comment:
I don't know this pattern, how does this relate to the c3d_scope already defined at line 96?

Reply:
This just initializes that static variable, which cannot be done in-class.